pkgs/servers/http/nginx/modules.nix

```diff
@@ -33,6 +33,38 @@ in
     inputs = [ pkgs.openssl ];
   };
 
+  auth-a2aclr = {
+    src = fetchFromGitLab {
+      name = "auth-a2aclr";
+      owner = "arpa2";
+      repo = "nginx-auth-a2aclr";
+      rev = "bbabf9480bb2b40ac581551883a18dfa6522dd63";
+      sha256 = "sha256-h2LgMhreCgod+H/bNQzY9BvqG9ezkwikwWB3T6gHH04=";
+    };
+    inputs =
+      let
+        inherit ((import (fetchFromGitHub {
+          owner = "ngi-nix";
+          repo = "arpa2";
+          rev = "c31b76bfc0139f59236224dc51bc28f262e77665";
+          sha256 = "sha256-eU7mRu+xoEwdW+zPkVWkxIfnziQX8QiXScor7d1oHD8=";
+        })).packages.x86_64-linux) arpa2common;
```

Comment:
We don't want to do that in nixpkgs. Please upstream the changes.

Reply:
Sure, I'll try to contact the devs.

Reply:
Also, this is IFD, and should be avoided in nixpkgs